spec/acceptance/helpers/repo_helper.rb

```diff
@@ -50,71 +50,10 @@ def install_puppet_repo(host)
-      # +reponame+: The base name of the repo, e.g. '6_X'
-      # +type+: Which repo to install:
-      #   :main for the main repo containing SIMP puppet modules
-      #   :deps for the SIMP dependency repo containing OS or application
-      #         RPMs not available from standard CentOS repos
-      #
-      # @fails if the specified repo cannot be installed on host
-      def install_internet_simp_repo(host, reponame, type)
-        case type
-        when :main
-          full_reponame = reponame
-          # FIXME: Use a gpgkey list appropriate for more than 6_X
-          repo = <<~EOM
-            [simp-project_#{reponame}]
-            name=simp-project_#{reponame}
-            baseurl=https://packagecloud.io/simp-project/#{reponame}/el/$releasever/$basearch
-            gpgcheck=1
-            enabled=1
-            gpgkey=https://raw.githubusercontent.com/NationalSecurityAgency/SIMP/master/GPGKEYS/RPM-GPG-KEY-SIMP
-                   https://download.simp-project.com/simp/GPGKEYS/RPM-GPG-KEY-SIMP-6
-            sslverify=1
-            sslcacert=/etc/pki/tls/certs/ca-bundle.crt
-            metadata_expire=300
-          EOM
-        when :deps
-          full_reponame = "#{reponame}_Dependencies"
-          # FIXME: Use a gpgkey list appropriate for more than 6_X
-          repo = <<~EOM
-            [simp-project_#{reponame}_dependencies]
-            name=simp-project_#{reponame}_dependencies
-            baseurl=https://packagecloud.io/simp-project/#{reponame}_Dependencies/el/$releasever/$basearch
-            gpgcheck=1
-            enabled=1
-            gpgkey=https://raw.githubusercontent.com/NationalSecurityAgency/SIMP/master/GPGKEYS/RPM-GPG-KEY-SIMP
-                   https://download.simp-project.com/simp/GPGKEYS/RPM-GPG-KEY-SIMP-6
-                   https://yum.puppet.com/RPM-GPG-KEY-puppetlabs
-                   https://yum.puppet.com/RPM-GPG-KEY-puppet
-                   https://apt.postgresql.org/pub/repos/yum/RPM-GPG-KEY-PGDG-96
-                   https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-$releasever
-            sslverify=1
-            sslcacert=/etc/pki/tls/certs/ca-bundle.crt
-            metadata_expire=300
-          EOM
-          full_reponame = "#{reponame}_Dependencies"
-        else
-          fail("install_internet_simp_repo() Unknown repo type specified '#{type.to_s}'")
-        end
-        puts('='*72)
-        puts("Using SIMP #{full_reponame} Internet repo from packagecloud")
-        puts('='*72)
-
-        create_remote_file(host, "/etc/yum.repos.d/simp-project_#{full_reponame.downcase}.repo", repo)
-      end
-
       # Set up SIMP repos on the host
       #
-      # By default, the SIMP '6_X' repos available from packagecloud
-      # will be configured.  This can be overidden with the BEAKER_repo
-      # environment variable as follows:
+      # By default, the SIMP '6_X' repos will be configured.  This can be
```

Comment:
Please update this documentation. 6_X does note make sense.  Also, if we want to test with unstable repos (pre-release check), not obvious how to do that